src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import covidReducer from './covid/covid';
+
+const reducer = combineReducers({
+  covid: covidReducer,
+
+});
+
+const store = createStore(reducer, applyMiddleware(thunk, logger));
+
```

Comment:
- [ ] Please do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data displayed on the browser console.